credentials/apps/verifiable_credentials/utils.py

```diff
@@ -39,8 +40,9 @@ def get_user_credentials_data(username, model):
                 credential.credential.program.authoring_organizations.values_list("name", flat=True)
             )
         elif model == "coursecertificate":
+            course_run = CourseRun.objects.filter(key=credential.credential.course_id).first()
             credential_uuid = credential.credential.course_id
-            credential_title = credential.credential.title
+            credential_title = course_run.course.title
```

Comment:
If such a CourseRun does not exist, we will get an AttributeError here.
Also, I think it is still worth taking `credential.credential.title` as a priority, and if it empty, then refer to `course_run.course.title`.